app/services/katello/pulp3/content_view_version/import.rb

```diff
@@ -61,12 +61,12 @@ def self.create_or_update_gpg!(organization:, params:)
           gpg
         end
 
-        def self.metadata_map(metadata, product_only: false, custom_only: false)
+        def self.metadata_map(metadata, product_only: false, custom_only: false, redhat_only: false)
```

Comment:
Would it make sense to have a method for dealing with custom product metadata vs red hat product metadata?

Reply:
Hmm I am not sure. Not averse to having separate methods as long as it doesn't have duplicated logic (its a one line change the methods are going to be identical).
Alternatively we could just ignore the skip content based on custom/redhat and have the caller filter that further ?